charts/pravega-operator/templates/certificate.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: selfsigned-cert
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: pravega-operator
+    app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+{{- if .Values.webhookCert.selfsigned }}
+  selfSigned: {}
+{{- else }}
+  ca:
+    secretName: pravega-operator-key-pair
```

Comment:
the names for all resources are generated in `_helpers.tpl` where the release name is prepended to the resource name.Could we follow that approach for this one well to ensure consistency?